frontend/src/pages/SignupPage/index.js

```diff
@@ -168,13 +191,26 @@ const SignupPage = () => {
   const buttonHandler = state.step === STEP_4 ? submitData : continueToNextStep;
 
   return (
-    <div className={styles.root}>
-      <div className={styles.logo}>
-        <Logo />
-      </div>
-      <div className={styles.main}>{stepInfo[state.step].component}</div>
-      <Button onClick={buttonHandler}>{stepInfo[state.step].buttonText}</Button>
-    </div>
+    <>
```

Comment:
When you have a component like `MainContainer` which encapsulates all the other elements, you don't need a fragment anymore `<>`